app/backend/src/tests/test_fixtures.py

```diff
@@ -639,7 +639,11 @@ def add_generic_rpc_handlers(self, generic_rpc_handlers):
 
         self.handlers.update(generic_rpc_handlers[0]._method_handlers)
 
-    def unary_unary(self, uri, request_serializer, response_deserializer):
+    def add_registered_method_handlers(self, service_name, method_handlers):
+        # Compatibility method for newer grpc versions
+        pass
```

Comment:
Could you clarify this please? What is it doing?

Reply:
Ooo okay on second look I think I don't need it. We bumped grpcio in #6871 and I forgot to pip install. I think there was some weirdness due to that causing the tests to fail.

I put it back as pip installing the latest requirements.txt seems to have fixed it.

I'm still wondering why that did make the tests pass now though ahah.